### PR TITLE
init update readme and keywords in packages

### PR DESCRIPTION
**Pull request by livingrockrises**
# Summary

I noticed either unsatisfactory content in some of the README files or outdated information (like missing account V2) so I have opened this PR to iteratively fix this. Feel free to contribute besides reviews! Ty 

Related Issue: # (issue number)

## Change Type

- [ ] Bug Fix
- [x] Refactor
- [ ] New Feature
- [ ] Breaking Change
- [ ] Documentation Update
- [ ] Performance Improvement
- [ ] Other

# Checklist

- [x] My code follows this project's style guidelines
- [x] I've reviewed my own code
- [x] I've added comments for any hard-to-understand areas
- [x] I've updated the documentation if necessary
- [x] My changes generate no new warnings
- [x] I've added tests that prove my fix is effective or my feature works
- [x] All unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published

# Additional Information

# Branch Naming

### CODE REVIEW

File: packages/account/Readme.md [hunk 1]
```diff
@@ -16,12 +16,14 @@ yarn add @biconomy/account
...
+Integrating and deploying Smart Accounts, building and sending user operations is a key offering of any toolkit designed for ERC4337. This package seamlessly integrates the essential features associated with ERC-4337 and simplifies the development of your Dapp's account and transaction rails with added usability features. 
 
-The Biconomy account package achieves this by providing a comprehensive set of methods that enable developers to effortlessly create UserOperations. Combined with the sophisticated backend infrastructure of the Biconomy platform, it ensures efficient and reliable transmission of these operations across EVM networks.
+Account package achieves this by providing a comprehensive set of methods that enable developers to effortlessly create UserOperations. Combined with the sophisticated, developer friendly and scalable infrastructure of Biconomy, it ensures efficient and reliable transmission of these operations across multiple EVM chains.
```
Comment (ankurdubey521) [inline packages/account/Readme.md:21]: "The" account?

File: README.md [hunk 2]
```diff
@@ -25,22 +25,45 @@ The Biconomy SDK is your all-in-one toolkit for building decentralized applicati
...
+import { providers } from 'ethers'
+import { ChainId } from "@biconomy/core-types"
+
+
+const module = await ECDSAOwnershipValidationModule.create({
+  signer: wallet,
+  moduleAddress: DEFAULT_ECDSA_OWNERSHIP_MODULE
+  })
+
+  let biconomySmartAccount = await BiconomySmartAccountV2.create({
```
Comment (ankurdubey521) [inline README.md:41]: indentation for let

**Comment by ankurdubey521** [comment]
Looks like linting and tests need to be fixed @livingrockrises

File: README.md (hunk 2 above)
Comment (livingrockrises) [reply README.md]: fixed and changed to const

**Comment by livingrockrises** [comment]
> Looks like linting and tests need to be fixed @livingrockrises

Ah I will change the base to development then. final PR will be going from development to master this week.

File: packages/web3-auth-native/.gitignore
Comment (AmanRaj1608) [inline packages/web3-auth-native/.gitignore:1]: all these files already don't exist in develop

File: README.md [hunk 3]
```diff
@@ -25,22 +25,45 @@ The Biconomy SDK is your all-in-one toolkit for building decentralized applicati
...
 Leverage standardized bundler infrastructure for efficient operation of account abstraction across EVM networks.
 
 ```javascript
+
+import { IBundler, Bundler } from '@biconomy/bundler'
+
+
 const bundler: IBundler = new Bundler({
-    bundlerUrl: '', // From Biconomy Dashboard
+    bundlerUrl: 'https://bundler.biconomy.io/api/v2/80001/nJPK7B3ru.dd7f7861-190d-41bd-af80-6877f74b8f44',
```
Comment (AmanRaj1608) [inline README.md:63]: Instead of this can add `<bundlerKey>` and a comment, `// Please go to https://dashboard.biconomy.io and generate bundler`.

**Review by AmanRaj1608 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (livingrockrises) [reply README.md:63]: done

**Review by ankurdubey521 — APPROVED** [review]

**Pull request merged**